lib/vmdb/plugins/asset_path.rb

```diff
@@ -0,0 +1,30 @@
+require_relative '../inflections'
+Vmdb::Inflections.load_inflections
+
+module Vmdb
+  class Plugins
+    class AssetPath
+      attr_reader :name
+      attr_reader :path
+      attr_reader :webpack_name
```

Comment:
This too me seems a bit too (frontend tech) specific, but I don't have an alternative at the moment.

Reply:
Maybe `underscore_name`?  `repo_name`?

Reply:
yeah, I couldn't come up with a better name... it's purpose is for webpack, hence why I chose it, but I agree it's a naming-coupling.  @himdel Thoughts?

Reply:
`repo` sounds reasonable too :)

The only thing it must match is the part before the slash in `= javascript_pack_tag 'manageiq-v2v/migration'`.

Reply:
how about `namespace`?  I'm concerned about `repo`, only because the actual git repo name may not match the engine's namespace  (e.g. miq_v2v_ui_plugin "repo", currently has the ManageIQ::V2V "namespace", which will convert to `manageiq-v2v`).  So, `namespace` feels a bit more accurate, IMO.